legal-api/src/legal_api/__init__.py

```diff
@@ -59,8 +59,9 @@ def create_app(run_mode=os.getenv("FLASK_ENV", "production"), **kwargs):
     babel.init_app(app)
     endpoints.init_app(app)
 
-    with app.app_context():  # db require app context
-        digital_credentials.init_app(app)
+    # TODO commented out Digital Credentials functionality for now.
+    # with app.app_context():  # db require app context
+    #     digital_credentials.init_app(app)
```

Comment:
commenting out digital credentials stuff for now.  DITP can look into this when they work on updating DC code to work with new model